### PR TITLE
Purge all remaining references to the baystation ported SolGov, and update the Sol Alliance flag

### PR DESCRIPTION
**Pull request by Generalcamo**
PR made in collaboration with and with the support of the Human Lore Team.

image

* Replaces the SolGov flag ported from Baystation to one more accurate to the canon flag
* Purges all references to SolGov where present, and replaces them with "Sol Alliance"
* Rewritten Sol Alliance citizenship description, written with assistance from DanseMacabre and approved by the Human Lore Team

These has been in for a number of years, however the Human Lore Team wishes to move away from the bayism of "SolGov", and this PR reflects this.

This may break existing loadouts that acquire the "SolGov" flag; they will need to replace it with the "Sol Alliance" flag.

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]
BLESSED BE SOL!!!

File: html/changelogs/generalcamo-solgovpurge.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+author: GeneralCamo
+
+delete-after: True
+
+changes:
+  - tweak: "Changed the in-game Sol Alliance flag to reflect the flag in canon."
+  - tweak: "Purged all references to "SolGov", as this is now considered a non-canon remnant from Baystation."
```
Comment (SleepyGemmy) [inline html/changelogs/generalcamo-solgovpurge.yml:7]:
```suggestion
  - tweak: "Purged all references to \"SolGov\", as this is now considered a non-canon remnant from Baystation."
```
escape characters.

File: html/changelogs/generalcamo-solgovpurge.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+author: GeneralCamo
+
+delete-after: True
+
+changes:
+  - tweak: "Changed the in-game Sol Alliance flag to reflect the flag in canon."
+  - tweak: "Purged all references to \"SolGov"\, as this is now considered a non-canon remnant from Baystation."
```
Comment (SleepyGemmy) [inline html/changelogs/generalcamo-solgovpurge.yml:7]:
```suggestion
  - tweak: "Purged all references to \"SolGov\", as this is now considered a non-canon remnant from Baystation."
```

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by Generalcamo** [comment]
Was unaware of the merge conflict. Working on it now.

**Pull request merged**